### DISCUSSION TITLE
能否上架Edge扩展商店

### DISCUSSION THREAD

**Discussion by Megadotzip** [riskers/github-plus-extension#4] [feature] [answered]
Chrome扩展商店被墙，还是Edge风方便些

**Comment by riskers** [comment]
@Megadotzip 没开发过 Edge 插件，我了解一下，如果和 Chrome 插件差异太大，可能不会有后续了。

**Comment by miranahf** [comment, marked as answer]
我试了一下，clone 项目进行编译，然后直接用 edge 扩展加载编译后的文件，是可以用的
